.github/workflows/fedora-copr-build.yml

```diff
@@ -6,26 +6,64 @@ on:
     # See https://docs.github.com/en/actions/reference/events-that-trigger-workflows#schedule
     - cron: "45 0 * * *"
 
-  workflow_dispatch: {}
+  workflow_dispatch:
+    inputs:
+      strategy:
+        description: 'build strategy'
+        required: true
+        type: choice
+        options:
+        - big-merge
+        - pgo
 
 jobs:
+  job1:
+    runs-on: ubuntu-latest
+    outputs:
+      mymatrix: ${{ steps.set-matrix.outputs.myoutput }}
+    steps:
+    - id: set-matrix
+      run: |
+        BIG_MERGE_CONFIG=$(jq -c <<EOF
+          {
+              "name": "big-merge",
+              "copr_project_tpl": "@fedora-llvm-team/llvm-snapshots-big-merge-YYYYMMDD",
+              "copr_target_project": "@fedora-llvm-team/llvm-snapshots",
+              "clone_url_tpl": "https://src.fedoraproject.org/rpms/PKG.git",
+              "clone_ref": "rawhide"
+          }
+        EOF
+        )
+
+        PGO_CONFIG=$(jq -c <<EOF
+          {
+            "name": "pgo",
+            "copr_project_tpl": "@fedora-llvm-team/llvm-snapshots-pgo-YYYYMMDD",
+            "copr_target_project": "@fedora-llvm-team/llvm-snapshots-pgo",
+            "extra_script_file": "scripts/functions-pgo.sh",
+            "clone_url_tpl": "https://src.fedoraproject.org/forks/kkleine/rpms/PKG.git",
+            "clone_ref": "pgo"
+          }
+        EOF
+        )
+
+        if [[ '${{ inputs.strategy }}' == 'big-merge' ]]; then
+          NAMES='["big-merge"]'
```

Comment:
Per https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/running-variations-of-jobs-in-a-workflow#example-adding-configurations I think just includes would be enough and we don't need to enumerate names separately?

Reply:
Addressed in 25a16dd